.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,77 @@
+name: 🐞 Bug Report
+description: Tell us about something that's not working the way we (probably) intend.
+labels: ["Platform: KMP", "Type: Bug"]
+body:
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Platform
+      description: Which platform do you use?
+      multiple: true
+      options:
+        - Apple
+        - Android
+        - Desktop
+    validations:
+      required: true
+
+  - type: dropdown
+    id: installed
+    attributes:
+      label: Installed
+      description: sentry-cocoa installed with (if applicable)
+      options:
+        - Swift Package Manager
+        - CocoaPods
+
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: Which version of sentry-kotlin-multiplatform do you use?
+      placeholder: 0.2.0 ← should look like this
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to Reproduce
+      description: How can we see what you're seeing? Specific is terrific.
+      placeholder: |-
+        1. foo
+        2. bar
+        3. baz
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Result
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Result
+      description: Logs? Screenshots? Yes, please.
+    validations:
+      required: true
+
+  - type: input
```

Comment:
I feel like this input is rather useless - if people are willing to submit a PR, they will just submit the PR instead of opening an issue imo :D